src/Npgsql/TypeHandlers/CompositeHandlers/MappedCompositeHandler.cs

```diff
@@ -116,31 +116,26 @@ public static MappedCompositeHandler<T> Create(PostgresCompositeType pgType, Con
                 var attr = clrMember.GetCustomAttribute<PgNameAttribute>();
                 var name = attr?.PgName ?? nameTranslator.TranslateMemberName(clrMember.Name);
 
-                int pgFieldIndex;
-                PostgresCompositeType.Field pgField = null;
-
-                for (pgFieldIndex = pgFields.Count - 1; pgFieldIndex >= 0; --pgFieldIndex)
+                for (var pgFieldIndex = pgFields.Count - 1; pgFieldIndex >= 0; --pgFieldIndex)
                 {
-                    pgField = pgFields[pgFieldIndex];
-                    if (pgField.Name == name) break;
+                    var pgField = pgFields[pgFieldIndex];
+                    if (pgField.Name == name)
```

Comment:
Optional nit: inverse condition to reduce nesting

Reply:
This is an intentional change to remove the check after the loop.

Reply:
I don't see a check... I just meant to do

```c#
if (pgField.Name != name)
    continue;

// rest of the code
```

Reply:
> I don't see a check...

It was there, but removed now.

> I just meant to do

Ah, I see. One moment...